src/renderer/components/MintInfoView.tsx

```diff
@@ -145,8 +182,14 @@ export function MintInfoView(props: { mintKey: string }) {
               );
             }}
           >
-            {mintAuthorityIsNull ? 'Mint closed' : 'Close Mint'}
-          </Button>
+            <div>
+              Are you sure you want to close the token mint. This will set the
```

Comment:
Copy nitpick, I'd write this as:

```
Are you sure you want to close the token mint?
This will set the update authority for the mint 
to null, and is not reversable.
```

and/or maybe drop the details and just change the latter part to just `This is not reversable`. Users don't usually read